src/routes/alerts/alerts.controller.spec.ts

```diff
@@ -178,12 +178,13 @@ describe('Alerts (Unit)', () => {
         );
 
         networkService.get.mockImplementation((url) => {
+          console.log(url);
           switch (url) {
             case `${safeConfigUrl}/api/v1/chains/${chain.chainId}`:
               return Promise.resolve({ data: chain });
-            case `${chain.transactionService}/api/v1/safes/${getAddress(
-              safe.address,
-            )}`:
+            case `${chain.transactionService}/api/v1/modules/${delayModifier}/safes/`:
```

Comment:
These changes are included in another PR if I'm not wrong (probably related to the conflict regarding this file).

Reply:
Rebasing fixed this.